tests/unit/test_toolchain.cpp

```diff
@@ -92,9 +104,11 @@ void TestToolchain(void)
     test_packed_enum();
 }
 
+#ifndef _WIN32
```

Comment:
That still doesn't fix the issue of having a `#ifndef _WIN32`, in every file. We can't have duplicate function definitions.